src/field.ts

```diff
@@ -0,0 +1,43 @@
+import Piece from './pieces/piece';
+
+class Field {
+    private _piece: Piece | null;
+    private _x: number;
+    private _y: number;
```

Comment:
Czy te podkreślenia mają jeszcze sens w TS, kiedy używamy "private"?

Reply:
Dlaczego zreygnowaliśmy z osobnego pliku types?
Takie były ustalenia na daily?

Reply:
> Czy te podkreślenia mają jeszcze sens w TS, kiedy używamy "private"?
W zasadzie to tak próbowałem ale pojawiała mi się informacja, że duplikuje nazwę zmiennej. Nie wiem jak to obejść :)

> Dlaczego zreygnowaliśmy z osobnego pliku types?
Takie były ustalenia na daily?
W types był Field i BasicPiece. Obydwa przeniosłem do oddzielnych plików. Nie wiem czy w TS praktykuje się dawanie interfejsów do oddzielnych plików. Zrobiłem to biorąc przykład jak to się robi w Javie. Nie ma problemu żeby przenieść BasicPiece do types.